spec/models/request_spec.rb

```diff
@@ -23,12 +24,27 @@
       let!(:request_started) { create(:request, :started) }
       let!(:request_fulfilled) { create(:request, :fulfilled) }
 
-      it "scopes" do
+      it "scopes by status" do
         expect(Request.status_pending).to eq([request_pending])
         expect(Request.status_started).to eq([request_started])
         expect(Request.status_fulfilled).to eq([request_fulfilled])
       end
     end
+
+    describe "#request_type" do
+      it "scopes by request_type" do
```

Comment:
This doesn't describe a scope, just the result of a method. But also I don't really see the value in this test - you're basically just checking that Rails works the way it's supposed to.